packages/salesforcedx-apex-replay-debugger/src/core/heapDumpService.ts

```diff
@@ -320,4 +377,19 @@ export class HeapDumpService {
   ) {
     return varContainer.name === extentValueEntry.keyDisplayValue;
   }
+
+  public isTriggerExtent(outerExtent: HeapDumpExtents) {
+    if (
+      (outerExtent.typeName === 'Boolean' ||
+        outerExtent.typeName.startsWith('List<') ||
+        outerExtent.typeName.startsWith('Map<')) &&
```

Comment:
And the reason we don't have `Set<` is because none of the trigger variables are sets, right (https://developer.salesforce.com/docs/atlas.en-us.apexcode.meta/apexcode/apex_triggers_context_variables.htm)?

Would it make sense to make this more future-proof and just add `outerExtent.typeName.startsWith('Set<')` as well?

Reply:
I had a quick chat with Mike Boilen on this, the List and Map for types would not be changing to Set. We're good here.